src/Commands/AccessCommands.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+namespace Drupal\access\Commands;
+
+use Drush\Commands\DrushCommands;
+use DateTime;
+use Drupal;
+
+/**
+ * Various utility commands for Access.
+ */
+class AccessCommands extends DrushCommands {
+    /**
+     * Ingest ACCESS organizations
+     *
+     * @command access:ingest-orgs
+     * @aliases ingest-organizations
+     * @options limit The number of records to process
+     */
+    public function ingestOrganizations($options = array('limit' => 100))
+    {
+        $orgs = array();
+
+        $path = \Drupal::service('file_system')->realpath("private://") . '/.keys/secrets.json';
+        if (!file_exists($path)) {
+            if ($options['verbose']) {
+                $this->output()->writeln('<error>Unable to find ACCESS API key file</error>');
+            }
+            return;
+        }
+        $secrets_json_text = file_get_contents($path);
+        $secrets_data = json_decode($secrets_json_text, true);
+        if (!isset($secrets_data['access_api_key']) || !$secrets_data['access_api_key']) {
```

Comment:
The api key is currently stored on the live filesystem with the 'ramps_api_key' key.